test/modules/collapseWhitespace.js

```diff
@@ -5,9 +5,12 @@ import maxPreset from '../../lib/presets/max';
 
 describe('collapseWhitespace', () => {
     const html = ` <div>
-            <b> hello  world! </b><a>other link
-	</a>
-	  </div>  `;
+    <p>      Hello world	</p>
```

Comment:
`<p>` is neither one of the `noTrimWhitespacesArroundElements` nor one of the `noTrimWhitespacesInsideElements`. The spaces within and around the `<p>` should be removed under `aggressive` mode.